jre_emul/android/platform/aosp/src/main/java/javax/net/ssl/X509ExtendedTrustManager.java

```diff
@@ -0,0 +1,234 @@
+/*
```

Comment:
Where is the repository for this package's sources? https://android.googlesource.com/ doesn't have a platform/aosp repository.

Reply:
Thanks for deleting these sources.